framework/src/mesh/GeneratedMesh.C

```diff
@@ -73,6 +73,43 @@ GeneratedMesh::GeneratedMesh(const InputParameters & parameters) :
 {
   if (_gauss_lobatto_grid && (_bias_x != 1.0 || _bias_y != 1.0 || _bias_z != 1.0))
     mooseError("Cannot apply both Gauss-Lobatto mesh grading and biasing at the same time.");
+
+  // All generated meshes are regular orthogonal meshes
+  _regular_orthogonal_mesh = true;
```

Comment:
a small question, is the mesh with TRI3 in 2D for example considered orthogonal mesh, or just QUAD4?